src/request/import/index.html

```diff
@@ -61,6 +74,24 @@ <h1 data-i18n="import-heading-log-in">Log in</h1>
             </div>
         </div>
 
+        <!-- privacy agent -->
+        <div id="privacy" class="page">
+            <div class="page-header">
+                <a tabindex="0" class="page-header-back-button icon-back-arrow"></a>
+                <h1 data-i18n="import-heading-log-in">Log in</h1>
+            </div>
+            <div class="page-body">
+                <div class="grow"></div>
+                <div class="agent"></div>
+            </div>
+        </div>
+
+        <!-- enter recovery words -->
+        <form id="words" class="page"></form>
+
+        <!-- choose key type -->
+        <form id="choose-key-type" class="page"></form>
```

Comment:
I'm generally more in favor of *not* putting whole pages as components, but leaving their markup and structure here in the `index.html` as HTML and only using *real* components like the `RecoverWords` component externally.